packaging/bin/java-home

```diff
@@ -73,40 +28,21 @@ validjre() {
 #
 if [ -n "${OVIRT_ENGINE_JAVA_HOME}" ]; then
 	if [ -n "${OVIRT_ENGINE_JAVA_HOME_FORCE}" ] || validjre "${OVIRT_ENGINE_JAVA_HOME}"; then
-		echo "${OVIRT_ENGINE_JAVA_HOME}"
-		exit 0
+		JAVA_HOME="${OVIRT_ENGINE_JAVA_HOME}"
 	else
+		echo "OVIRT_ENGINE_JAVA_HOME (${OVIRT_ENGINE_JAVA_HOME}) is not a valid JRE" >&2
 		exit 1
 	fi
 fi
 
-#
-# select alternates folder
-# if we find, first wins
-#
-DIRS=
-case "${COMPONENT}" in
-	engine)
-		DIRS="/usr/lib/jvm/java-11"
-	;;
-	reports)
-		DIRS="/usr/lib/jvm/jre-1.7.0 /usr/lib/jvm/jre-1.7.0_openjdk /usr/lib/jvm/jre"
-	;;
-esac
-for dir in ${DIRS};  do
-	if validjre "${dir}"; then
-		echo "${dir}"
-		exit 0
-	fi
-done
-
-# Java 11 has no jre/jdk, just one folder.
-JAVA_HOME=""
-for dir in /usr/lib/jvm/*; do
-	if validjre "${dir}"; then
-		JAVA_HOME="${dir}"
+ALTERNATIVES_OPENJDK_JRE=$(alternatives --list 2>/dev/null | grep jre_openjdk)
```

Comment:
We could eventually just do the following:
`dirname $(dirname $(readlink -f $(which java)))`

This finds the JAVA_HOME for the current 'java'